core/libp2p/multi/multibase_codec/codecs/base64.cpp

```diff
@@ -78,6 +78,13 @@ namespace {
   }
 }  // namespace
 
+OUTCOME_CPP_DEFINE_CATEGORY(libp2p::multi::detail, Base64DecodeError, e) {
+  switch(e) {
+    case libp2p::multi::detail::Base64DecodeError::kInvalidInput:
+      return "Input is not a valid base64 string";
+  }
```

Comment:
handle default case.

Reply:
well, you can actually combine all enums as `enum class BaseError`, and just provide list of errors there. Then, use shared error code for all encoders/decoders.